examples/12-fluids/03-fluids_isosurface.py

```diff
@@ -87,18 +82,19 @@
     opacity=0.3,
 )
 
-for i in range(number_of_iso_surface):
-    iso_surface = dpf.operators.mesh.mesh_cut(
-        field=P_S[0], iso_value=min_pressure, closed_surface=0, mesh=whole_mesh, slice_surfaces=True
-    ).eval()
+vec_iso_surfaces = [-153.6, -100.0, -50.0, 50.0, 100.0, 150.0, 200.0, 300.0, 361.8]
+
+iso_surfaces = dpf.operators.mesh.iso_surfaces(field=P_S[0], mesh=whole_mesh, slice_surfaces=True,
+                                              vector_iso_values=vec_iso_surfaces).eval()
+
+for i in range(len(iso_surfaces)):
     P_S_step = dpf.Field(location=dpf.locations.overall, nature=dpf.common.natures.scalar)
-    P_S_step.append([min_pressure], i)
+    P_S_step.append(vec_iso_surfaces[i], i)
     P_S_step.name = "static pressure"
     P_S_step.unit = "Pa"
```

Comment:
P_S_step doesn't need to be created by hand, as the second output of the operator should be giving it